builder/lxc/config.go

```diff
@@ -65,6 +73,10 @@ func NewConfig(raws ...interface{}) (*Config, error) {
 		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Failed parsing init_timeout: %s", err))
 	}
 
+	if c.LxcTemplate.Name != "" && c.RootFs != (RootFsConfig{}) {
+		errs = packer.MultiErrorAppend(errs, fmt.Errorf("Cannot build with both lxc_template and rootfs configuration options"))
```

Comment:
The thing with the none template is that the config will also be totally empty. All the none template does is create a dir and add a config file with only network config. I'm not sure this will simplify the plugin.